.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
Consider adding a comment or documentation that explains the purpose of these variables and how they should be set, especially if this code will be shared or used by other developers.